tests/testthat/test-carehomes-multistrain.R

```diff
@@ -517,38 +790,62 @@ test_that("Can calculate Rt with an empty second variant ", {
 
   initial <- carehomes_initial(mod$info(), 10, p)
   mod$set_state(initial$state, initial$step)
-  index <- mod$info()$index$S
+  index_S <- mod$info()$index$S
+  index_R <- mod$info()$index$R
 
   end <- sircovid_date("2020-05-01") / p$dt
   steps <- seq(initial$step, end, by = 1 / p$dt)
 
   set.seed(1)
-  mod$set_index(index)
+  mod$set_index(c(index_S, index_R))
   y <- mod$simulate(steps)
 
-  rt_1_single_class <- carehomes_Rt(steps, y[, 1, ], p)
-  rt_all_single_class <- carehomes_Rt_trajectories(steps, y, p)
+  S <- y[1:19, , ]
+  R <- y[20:38, , ]
+
+  rt_1_single_class <- carehomes_Rt(steps, S[, 1, ], p, R = R[, 1, ])
+  rt_all_single_class <- carehomes_Rt_trajectories(steps, S, p, R = R)
+
+  expect_equal(rt_1$step, rt_1_single_class$step)
+  expect_equal(rt_1$date, rt_1_single_class$date)
+  expect_equal(rt_1$beta, rt_1_single_class$beta)
+  expect_equal(rt_1$eff_Rt_all, rt_1_single_class$eff_Rt_all)
+  expect_equal(rt_1$eff_Rt_general,
+               rt_1_single_class$eff_Rt_general)
+  expect_equal(rt_1$Rt_all, rt_1_single_class$Rt_all)
+  expect_equal(rt_1$Rt_general, rt_1_single_class$Rt_general)
+
+  expect_equal(rt_all$step, rt_all_single_class$step)
+  expect_equal(rt_all$date, rt_all_single_class$date)
+  expect_equal(rt_all$beta, rt_all_single_class$beta)
+  expect_equal(rt_all$eff_Rt_all,
+               rt_all_single_class$eff_Rt_all)
+  expect_equal(rt_all$eff_Rt_general,
+               rt_all_single_class$eff_Rt_general)
+  expect_equal(rt_all$Rt_all, rt_all_single_class$Rt_all)
+  expect_equal(rt_all$Rt_general,
+               rt_all_single_class$Rt_general)
 
-  expect_equal(rt_1, rt_1_single_class)
-  expect_equal(rt_all, rt_all_single_class)
 })
 
 
 test_that("Can calculate Rt with a second less infectious variant", {
   ## seed with 10 cases on same day as other variant
-
   p <- carehomes_parameters(sircovid_date("2020-02-07"), "england",
                             strain_transmission = c(1, 0.1),
                             strain_seed_date =
                               sircovid_date(c("2020-02-07", "2020-02-08")),
-                            strain_seed_rate = c(10, 0))
+                            strain_seed_rate = c(10, 0),
+                            strain_rel_severity = c(1, 0.8),
```

Comment:
I would keep the severity for another test and keep it to the default c(1,1) in this one